ecommerce/products/base.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+
+class Rarity:
+    COMMON = 'common'
+    UNCOMMON = 'uncommon'
+    RARE = 'rare'
+    ULTRA_RARE = 'ultra rare'
+    AMAZING_RARE = 'amazing rare'
+    SECRET_RARE = 'secret rare'
+
+class Languages:
+    ENGLISH = 'English'
+    JAPANESE = 'Japanese'
+    FRENCH = 'French'
+    SPANISH = 'Spanish'
+    GERMAN = 'German'
+    CHINESE ='Chinese'
+    KOREAN = 'Korean'
```

Comment:
You reckon this covers the main languages that will be used? I think you're right that we won't want every language. You reckon this covers 99% of cards?

Reply:
Yeah i was thinking the exact same as you for this. Like am I covering them all but I had a good think about it and I haven't seen cards in languages other than these so I'd say it covers 99% yeah.